src/component/JoinList.js

```diff
@@ -70,7 +73,7 @@ const JoinList = ({ firestore }) => {
       <Link to="/" className="button-link" id="newListButton">
         Start a New List
       </Link>
-    </p>
+    </React.Fragment>
```

Comment:
Just a heads up - I think we did a paragraph here on advice from the mentors. Might not still apply though - I thought I saw somewhere you were saying something about it causing a warning?

Reply:
🤔Nesting a `<form>` inside a `<p>` is invalid HTML. In general, if a tag isn't needed for semantic reasons, it's better to omit it